common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -693,6 +693,7 @@ def test_cohorts_management_a11y(self):
         self.cohort_management_page.a11y_audit.config.set_rules({
             "ignore": [
                 'aria-valid-attr',  # TODO: LEARNER-6611 & LEARNER-6865
+                'region',
```

Comment:
As with the other PR, having some sort of tracking ticket would be awesome, if you need help or want to coordinate on that let me know. We can use the Product (PROD) Jira Project to collect these.

Reply:
Totally agree, but wanted some feedback first. Mainly on the best way to break them up, e.g. by ignore rule, by test, etc.

Reply:
My first thought is to create tickets based on ignore rules, but I don't know how many hits are being triggered for each problematic rule.
@fysheets Axe rules are unlikely to produce false positives, as that has been one of their design goals in recent years, but I assume (without much confidence) that the ones that are currently throwing errors are probably fairly low-impact.  I'd still like to get all of this caught up nonetheless.

Reply:
I think ticket that is ignore by rule, then the comment would be to the same ticket throughout, so when a rule is fixed it can be fixed across the board.